src/diffusers/models/transformers/transformer_chronoedit.py

```diff
@@ -137,7 +137,11 @@ def apply_rotary_emb(
                 dropout_p=0.0,
                 is_causal=False,
                 backend=self._attention_backend,
-                parallel_config=self._parallel_config,
+                # FIXME(DefTruth): Since the key/value in cross-attention depends
```

Comment:
```suggestion
                # Reference: https://github.com/huggingface/diffusers/pull/12660
                # Since the key/value in cross-attention depends
```